bazel/external/wamr.BUILD

```diff
@@ -0,0 +1,23 @@
+licenses(["notice"])  # Apache 2
+
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+cmake(
+    name = "libiwasm",
+    cache_entries = {
+        "WAMR_BUILD_AOT": "0",
```

Comment:
Please match it to options used in Envoy. At the very least, please add:
```
"WAMR_BUILD_INTERP": "1",
"WAMR_BUILD_JIT": "0",
```
or does JIT work already?

Reply:
:hand: just curious, I didn't find any WAVM related BUILD file. Does it mean *WAVM* can not be linked in this repo yet?

Reply:
Correct, currently you cannot build WAVM in this repo. We're slowly fixing this (see: https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/160), which is why it would be helpful if WAMR worked.